benchopt/cli/tests/test_cli.py

```diff
@@ -248,10 +248,11 @@ def test_profiling(self, test_env_name):
                      '-n', '1', '-r', '1', '-o', SELECT_ONE_OBJECTIVE,
                      '--profile', '--no-plot'],
                     'benchopt', standalone_mode=False)
-
+        python_pgd_path = os.path.join(DUMMY_BENCHMARK_PATH, 'solvers',
+                                       'python_pgd.py')
```

Comment:
If possible, avoid using `os.path` and use `pathlib` everywhere.
```suggestion
        python_pgd_path = (
            DUMMY_BENCHMARK_PATH / 'solvers' / 'python_pgd.py'
        )
```